userspace/falco/app_actions/load_rules_files.cpp

```diff
@@ -100,9 +100,7 @@ application::run_result application::load_rules_files()
 
 		res = m_state->engine->load_rules_file(filename);
 
-		// Print the full output if verbose is true
-		if(m_options.verbose &&
-		   (!res->successful() || res->has_warnings()))
+		if((!res->successful() || res->has_warnings()))
```

Comment:
```suggestion
		if(!res->successful() || (m_options.verbose && res->has_warnings()))
```
Not sure, but wasn't the previous logic like this before? https://github.com/falcosecurity/falco/blob/4c4ed56c19b9a64287715963e04ccd30977aa327/userspace/engine/falco_engine.cpp#L209
If we have an error, print it. Else if there are warnings and we're verbose, print them.

Reply:
Yeah, you're right. I will update.